### ISSUE TITLE
onScroll never gets to 0 when scrolling to the top

### ISSUE THREAD

**Issue by tpae** [inovua/reactdatagrid#61]
- what edition are you using - enterprise
- version for `@inovua/reactdatagrid-community`or `@inovua/reactdatagrid-enterprise` - 4.0.29

Relevant code or config

```typescript
const GridScrollDemo: React.FC<TypeComputedProps> = (props) => {
  const [gridRef, setGridRef] = React.useState<TypeComputedProps>();

  const handleGridScroll = React.useCallback(() => {
    console.log('grid', gridRef?.getScrollTop()); // <--- THIS NEVER GETS TO ZERO
  }, [gridRef]);

  const handleOnReady = (
    ref: React.MutableRefObject<TypeComputedProps | null>
  ) => {
    if (ref.current) {
      setGridRef?.(ref.current);
    }

    props.onReady?.(ref);
  };

  return (
    <ReactDataGrid
      {...props}
      onReady={handleOnReady}
      onScroll={handleGridScroll}
    />
  );
};

```

What you did:

Trying to get the scroll value in order to synchronize scrolling with other components. I've also tried activating "undocumented" feature:
```jsx
    <ReactDataGrid
      {...props}
      onReady={handleOnReady}
      onScroll={handleGridScroll}
      scrollProps={{
        scrollDebounceDelay: 0
      }}
    />
```

I'm afraid it could cause other side effects, but I need this in order to synchronize the component scrolling.

https://github.com/inovua/reactdatagrid/blob/d7c1e7d011f38c2969c5add4e462485dc0853701/community-edition/hooks/useScrollProps.ts#L26

Problem description:

Since it never goes to 0, the synchronization is off.

Suggested solution:

Fix the bug to return scroll 0

**Comment by inovua-admin** [comment]
Will be included in the next version due today

**Comment by inovua-admin** [comment]
Fixed in `4.0.32`
